README.md

```diff
@@ -6,63 +6,152 @@
 ![PyPI - Downloads](https://img.shields.io/pypi/dm/pqm)
 [![arXiv](https://img.shields.io/badge/arXiv-2402.04355-b31b1b.svg)](https://arxiv.org/abs/2402.04355)
 
-Implementation of the PQMass two sample test from Lemos et al. 2024 [here](https://arxiv.org/abs/2402.04355)
+<!-- Implementation of the PQMass two sample test from Lemos et al. 2024 [here](https://arxiv.org/abs/2402.04355) -->
+
+[PQMass](https://arxiv.org/abs/2402.04355) is a new sample-based method for evaluating the quality of generative models as well assessing distribution shifts.
```

Comment:
The point about generative models is good, but you might want to add some more to mention that the method works for comparing any two samples to see if they come from the same distribution